community/lexicon/actor/profile.json

```diff
@@ -0,0 +1,97 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.actor.profile",
+  "description": "Extended profile information for AT Protocol actors",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "An extended profile record.",
+      "key": "literal:self",
+      "record": {
+        "type": "object",
+        "required": [
+          "displayName",
+          "createdAt"
+        ],
+        "properties": {
+          "displayName": {
+            "type": "string",
+            "maxGraphemes": 64,
+            "maxLength": 640
+          },
+          "description": {
+            "type": "string",
+            "description": "Free-form profile description text.",
+            "maxGraphemes": 256,
+            "maxLength": 2560
+          },
+          "avatar": {
+            "type": "blob",
+            "description": "Profile image",
+            "accept": [
+              "image/png",
+              "image/jpeg"
+            ],
+            "maxSize": 1000000
+          },
+          "banner": {
+            "type": "blob",
+            "description": "Banner image",
+            "accept": [
+              "image/png",
+              "image/jpeg"
+            ],
+            "maxSize": 1000000
+          },
+          "email": {
+            "type": "string",
+            "format": "email",
+            "maxLength": 255
+          },
+          "phone": {
+            "type": "string",
+            "maxLength": 20,
+            "description": "International format phone number"
+          },
+          "address": {
+            "type": "ref",
+            "ref": "community.lexicon.locations.address"
+          },
+          "links": {
```

Comment:
Similarly, links could be a useful lexicon on its own, so that we don't end up with slight variations on field names or constraints everywhere they may manifest if we didn't have a dedicated type

Reply:
Makes sense. I'll attach to this PR.